models/CreateNewPatientRequest.js

```diff
@@ -0,0 +1,83 @@
+/*
+
+Patient {
+
+  family : string
+  given : string
+  suffix : string
+  phone : {
```

Comment:
We should be able to accommodate 2 instances of phone number:  primary and secondary.
We only need this for the household registration, so I'm assuming we can just leave "secondary" blank for individual patient registration.

Reply:
The phone field takes a list which handles this. I've updated the file to reflect this.